test/functional/adb-commands-e2e-specs.js

```diff
@@ -2,12 +2,17 @@ import chai from 'chai';
 import chaiAsPromised from 'chai-as-promised';
 import 'mochawait';
 import ADB from '../../lib/adb.js';
+import path from 'path';
 
 chai.use(chaiAsPromised);
 // change according to CI
 const apiLevel = '21',
       IME = 'com.example.android.softkeyboard/.SoftKeyboard',
-      defaultIME = 'com.android.inputmethod.latin/.LatinIME';
+      defaultIME = 'com.android.inputmethod.latin/.LatinIME',
+      contactManagerPath = path.resolve(__dirname, '..', '..', '..', 'test',
```

Comment:
Here: https://github.com/appium/appium-adb/commit/353cbf21c97ed253243f8c855656f9610ad0964e
It's just when you use transpile on the fly (for coverage), the build repo is not used, so you get one less `..` in the PATH.